x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/azureopenai/AzureOpenAiAccount.java

```diff
@@ -9,7 +9,6 @@
 
 import org.elasticsearch.common.settings.SecureString;
 import org.elasticsearch.core.Nullable;
-import org.elasticsearch.xpack.inference.services.azureopenai.embeddings.AzureOpenAiEmbeddingsModel;
```

Comment:
Actually can we remove the whole class now 🤔 ? Doesn't look like it's used anymore right?

Reply:
Good point! Adjusted with [Remove unused class](https://github.com/elastic/elasticsearch/pull/108356/commits/84c507e9892a0b4545ebac700badb1602f82032f)